src/TcBlack/ObjectDefinition.cs

```diff
@@ -124,25 +124,19 @@ private TcObject TokenizeFunctionBlock()
                     extends = true;
                     implements = false;
                 }
-                else if (part.ToLower() == "abstract" || part.ToLower() == "final")
+                else if (part.ToLower() == "abstract"
+                    || part.ToLower() == "final"
+                    || part.ToLower() == "internal"
+                    || part.ToLower() == "public")
```

Comment:
```suggestion
                else if (
                    part.ToLower() == "abstract"
                    || part.ToLower() == "final"
                    || part.ToLower() == "internal"
                    || part.ToLower() == "public"
                )
```

Reply:
Sorry for the small changes, but if your building a code formatter the code of the formatter itself should also be consistent :p.